frameworks/TypeScript/bun/src/headers.ts

```diff
@@ -0,0 +1,23 @@
+export const headerPlainText = {
+  status: 200,
+  headers: {
+    'Content-Type': 'text/plain',
+    Server: 'bun',
+  },
+};
+
+export const headerJSON = {
+  status: 200,
+  headers: {
```

Comment:
try creating these Header objects

you can skip the status code as it is 200 by default